google/pubsub/v1/pubsub_gapic.yaml

```diff
@@ -29,6 +29,8 @@ collection_oneofs:
   - deleted_topic
 interfaces:
 - name: google.pubsub.v1.Subscriber
+  lang_doc:
+    java: To retrieve messages from a subscription, see the Subscriber class.
```

Comment:
I seem to not able to do `{@link Subscriber}` as something is escaping it. I'm not sure where though :cry: